ci-operator/config/openshift/installer/openshift-installer-master__altinfra.yaml

```diff
@@ -510,6 +510,22 @@ tests:
       FEATURE_SET: CustomNoUpgrade
     workflow: openshift-e2e-vsphere-multi-vcenter
   timeout: 6h0m0s
+- always_run: false
+  as: e2e-external-aws
+  optional: true
+  steps:
+    cluster_profile: aws
+    workflow: openshift-e2e-external-aws
+  timeout: 6h0m0s
+- always_run: false
+  as: e2e-external-aws-ccm
+  optional: true
+  steps:
+    cluster_profile: aws
+    env:
+      PLATFORM_EXTERNAL_CCM_ENABLED: "yes"
+    workflow: openshift-e2e-external-aws
+  timeout: 6h0m0s
```

Comment:
@r4f4 I am not sure if we'll need this jobs in `altinfra` as provisioning steps from this workflow is pure UPI. I am adding here just for consistency, but it sounds duplicated.

Also just realized that I missed the feature gate env vars. I will wait your feedback to make the changes, and not kill the existing jobs.

Reply:
If it doesn't exercise the capi provisioning, we don't need this in the __altinfra config. Now that capi is enabled in the "normal" installer, this config will probably be removed once we remove terraform from the Installer.

Reply:
ACK, I will remove in the next review cycle.